tests/phpunit/tests/hooks/wpHook/removeFilter.php

```diff
@@ -86,6 +86,40 @@ public function test_remove_filter_with_another_at_different_priority() {
 		$this->check_priority_exists( $hook, $priority + 1, 'Should priority of 3' );
 	}
 
+	/**
+	 * @ticket 17817
+	 *
+	 * This specificaly addresses the concern raised at
```

Comment:
I am shocked and appalled that this typo has remained. I'm going to make a cup of tea before I continue reviewing as punishment.